modules/change-collection-method.adoc

```diff
@@ -6,7 +6,7 @@
 = Changing the collection method
 
 [role="_abstract"]
-If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is set to `CORE_BPF` before you upgrade, if you are upgrading from 4.1 or later. Otherwise, set the collection method to `EBPF`. To set the collection method to `EBPF`, you must set the `forceCollection` parameter to `true` after the upgrade and make sure that the collection method is `EBPF`.
+If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is set to `CORE_BPF` before you upgrade.
```

Comment:
Do we need to mention EBPF (I don't think so?) If we do then just "The `EBPF` option is supported for version 4.4 and earlier." would be OK, I guess.